pages/index.html

```diff
@@ -4,9 +4,42 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link href="../stylesheets/index.css" rel="stylesheet" />
+  <title>Home</title>
 </head>
 <body>
-  
+  <header>
+    <nav>
+      <ul>
+        <li><a href="index.html">Home</a></li>
+        <li><a href="about.html">About</a></li>
+        <li><a href="portfolio.html">Portfolio</a></li>
+      </ul>
+    </nav>
+    
+  </header>
+  <main>
```

Comment:
I like how you added your photo next to the text here! You could've made the div a section tag for the photo.